app/components/post/index.js

```diff
@@ -84,6 +85,8 @@ function makeMapStateToProps() {
             isFlagged: isPostFlagged(post.id, myPreferences),
             isCommentMention,
             isLandscape: isLandscape(state),
+            previousPostExists: Boolean(previousPost),
+            beforePrevPostUser: (beforePrevPost ? beforePrevPost.user_id : null),
```

Comment:
Should the name of this variable maybe indicate it contains an ID for easier reading and to be consistent with other variable names like 'currentUserId` above?
```suggestion
            beforePrevPostUserId: (beforePrevPost ? beforePrevPost.user_id : null),
```

Reply:
Great suggestion, I made these changes, thank you!